src/framework/acl/builder/builder_functions.clj

```diff
@@ -1,42 +1,51 @@
 (ns framework.acl.builder.builder-functions)
 
+
```

Comment:
please  remove these formatting changes, only include code changes. we can create another PR for formatting